packages/compass-connections-navigation/src/navigation-item.tsx

```diff
@@ -258,7 +258,10 @@ export function NavigationItem({
           isExpandVisible={item.isExpandable}
           isExpandDisabled={
             item.type === 'connection' &&
-            item.connectionStatus === 'disconnected'
+            (item.connectionStatus === 'initial' ||
```

Comment:
I'm wondering if it isn't more future-proof to just do !== 'connected' or similar.

Reply:
By the looks of it this is missing connecting, for example? Or should it be enabled when connecting?

Reply:
Good point, I think what we want is to only disable expand when you are not connected, so reverting this logic sounds good

Reply:
Changed in https://github.com/mongodb-js/compass/pull/6123/commits/aeaa0de7c67b175f3ed54afb3586da59503e96bc while updating some sidebar tests